src/Entities/SpecificProjectEntity.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace ProjectManager\Entities;
+
+class SpecificProjectEntity extends ProjectEntity implements \JsonSerializable
+{
+    private string $client_name;
+    private string $client_logo;
+    private array $users = [];
+
+    public function setUsers(array $users): bool
+    {
+        foreach ($users as $user)
+        {
+            if($user instanceof UserEntity) {
+                $this->users[] = $user;
+            } else {
+                throw new \Exception ('Array contents are not correct');
```

Comment:
can we have a more specific message? what should they be? this doesnt really help me understand what iv done wrong